koji/collection-mash-split.py

```diff
@@ -160,6 +168,31 @@ def createrepo(self, path, comps=None, checksum=None):
         self.logger.debug("running %s" % cmd)
         kobo.shortcuts.run(cmd)
 
+    def inject_modulemd_yml(self, path, modulemd_yaml, modulemd_version=0):
+        """Generate modular metadata and inject it.
+
+        @param path: path to repository
+        @type path: str
+        @param modulemd_yaml: metadata template
+        @type modulemd_yaml: str
```

Comment:
Should `modulemd_version` also be documented?

Reply:
yes!